src/client/cl_input.c

```diff
@@ -1046,6 +1046,8 @@ void CL_FinishMove(usercmd_t *cmd)
 	}
 }
 
+#define MASK_CGAMEFLAGS_SERVERTIMEDELTA 0b00000110
```

Comment:
Binary literals are not part of C standard yet FYI. This is a gnu/clang extension if it works.

Reply:
Yeah it does work and I personally like the clarity/verbosity of it but I'll just make it a hex and an end of line comment to get the same effect within the standard.